src/Compatibility/Microsoft.DotNet.ApiCompatibility/DiagnosticBag.cs

```diff
@@ -73,6 +81,6 @@ public void Add(T difference)
         /// <summary>
         /// A list of differences contained in the diagnostic bag.
         /// </summary>
-        public IEnumerable<T> Differences => _differences;
+        public IList<T> Differences => _differences;
```

Comment:
Why do you need it to be an `IList`? The point of this is for consumers to use the `Add` APIs.

Reply:
Yeah i missed to revert it